.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
```

Comment:
the comment doesn't match the implementation, I think this makes all files  

Maybe just copy the core list in:
https://github.com/jenkinsci/jenkins/blob/master/.gitattributes

although may not need all of them as this repo won't have them

see also https://stackoverflow.com/a/38017715 which recommends not to just have `* text=auto` as it may auto detect an image type wrong

Reply:
Done, just used the same list as core.